doc/develop/extending.rst

```diff
@@ -330,6 +330,46 @@ To read an acquisition file you can use code such as:
     print das[0].metadata
 
 
+Starting odemis from the terminal/console
+=================================================
+
+After setting up the development environment it is possible to start odemis via the terminal.
```

Comment:
We have two types of config files: odemis.conf and the YAML file. To distinguish them clearly, normally, we call the first one the "configuration file" and the second one the "microscope file".
=> a couple of places should be updated to follow this rule.